config/crds/v1/patches/ent-patches.yaml

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
nit:  not from your PR, `ent-patches.yaml`, `ems-patches.yaml` and `apm-kibana-patches.yaml` have the same content, maybe we can have a single `remove-pod-template.patch.yaml`